query/test/fixtures/fast_query_expected_results.json

```diff
@@ -0,0 +1,125 @@
+{
+  "expected_result": [
```

Comment:
nit you could change before merging: why wrap the object below when the dict has only one key?

Reply:
fixed